### PR TITLE
Add link and default execute engine section

### PR DESCRIPTION
**Pull request by TianlongLiang**

### CODE REVIEW

File: content/en/blog/introduction_to_wamr_running_modes/index.md [hunk 1]
```diff
@@ -39,7 +39,11 @@ When you use WAMR in your products, there are a few things that should be consid
 
 ### Compile the engines
 
-All the WAMR execution engines can be built into single software at any combination with the limits that the fast interpreter and classic can't co-exist in a software binary, and the JIT has dependence on the interpreter. WAMR offers several CMake cache variables for the compilation control of the execution engines:
+Except that the Fast Interpreter can't co-exist with other execution engines in a software binary, all other execution engines can be built into single software at any combination.
+
+> Note: Classic Interpreter, as a foundation for LLVM JIT and Fast JIT, will be automatically included if you choose to include one of JIT execution engines
```
Comment (xwang98) [inline content/en/blog/introduction_to_wamr_running_modes/index.md]: cmake里不加WAMR_BUILD_INTERP也可以？

Comment (TianlongLiang) [reply content/en/blog/introduction_to_wamr_running_modes/index.md]: 会自动帮着开

File: content/en/blog/introduction_to_wamr_running_modes/index.md [hunk 2]
```diff
@@ -53,7 +57,12 @@ The example of CMake build command below compiles the Classic Interpreter, LLVM
   ```sh
   cmake -DWAMR_BUILD_INTERP=1 -DWAMR_BUILD_JIT=1 -DWAMR_BUILD_FAST_JIT=1 -B build
   ```
- 
+
+**The default Execution Engine when compiling multiple Engines into binary:**  
+
+1. When you compile all three execution engines: the JIT layers tier-up on the fly feature is automatically on; firstly, it starts on Fast JIT mode, then gradually switch to more efficient LLVM JIT mode.
```
Comment (xwang98) [inline content/en/blog/introduction_to_wamr_running_modes/index.md]: move this under "3. Compiled default running mode"

File: content/en/blog/introduction_to_wamr_running_modes/index.md [hunk 3]
```diff
@@ -122,7 +130,7 @@ wasm_runtime_call_wasm(exec_env, b_func, 1, wasm_argv);
 
 ## Try out iwasm from WAMR binary release
 
-If you want to have a quick try, it would be a good option to download the WAMR binary release and start the iwasm command.
+If you want to have a quick try, it would be a good option to download the WAMR from [binary release](https://github.com/bytecodealliance/wasm-micro-runtime/releases) and start the iwasm command.
 
 There are four command line options to control the running modes of iwasm:
 - `--interp`: run iwasm in class interpreter mode
```
Comment (xujuntwt95329) [inline content/en/blog/introduction_to_wamr_running_modes/index.md]:
```suggestion
- `--interp`: run iwasm in interpreter mode
```

**Pull request merged**